control_panel_api/k8s_patch.py

```diff
@@ -0,0 +1,45 @@
+import base64
+import datetime
+import json
+from datetime import timezone
+
+import kubernetes
+from kubernetes.config.kube_config import _is_expired
+
+
+def load_token(self):
+    if 'auth-provider' not in self._user:
+        return
+
+    provider = self._user['auth-provider']
+
+    if ('name' not in provider
+            or 'config' not in provider
+            or provider['name'] != 'oidc'):
+        return
+
+    parts = provider['config']['id-token'].split('.')
+
+    if len(parts) != 3:  # Not a valid JWT
+        return None
+
+    jwt_attributes = json.loads(
+        base64.b64decode(parts[1] + '==').decode('utf-8')
```

Comment:
This is the key part of this patch - the official client code is missing the ` + '=='` and so base64 complains about `Invalid padding`